### ISSUE TITLE
Error on joining a room which doesn't exist (BE)

### ISSUE THREAD

**Issue by wowawiwa** [abustany/goretro#14]
Currently, there is no way for the FE to know distinguish if the join-room request is taking time or if the room doesn't exist.

**Comment by abustany** [comment]
Yeah we clearly need a pattern here to deal with commands that can fail... Currently a command being "successful" just means it's been successfully received by the server, but not that it was successfully processed by the retro "object". The retro will currently just ignore invalid commands, but we need to be better at surfacing those errors. But then do we need to identify which command caused an error? If so do we need (optional) command IDs akin to the `id` field in JSON-RPC calls?
